packages/styled/types/tests.tsx

```diff
@@ -86,6 +86,7 @@ const Button = styled.button``
 const Input = styled.input`
```

Comment:
could you also add one a little bit simpler test that would test this thing alone? this one is testing that `Label` can be interpolated as a part of the selector